docs/content/languages/nodejs.md

```diff
@@ -0,0 +1,420 @@
+---
+title: "Fission functions with Nodejs"
+weight: 10
+---
+
+Fission supports functions written in Nodejs. Current fission nodejs runtime environment supports node version greater than 7.6.0. In this usage guide we'll cover how to use this environment, write functions, and work with dependencies.
+
+## Before you start
+
+We'll assume you have Fission and Kubernetes setup. If not, head over to the [install guide](../installation/_index.en.md).  Verify your Fission setup with:
+
+```
+fission --version
+```
+
+## Add the Nodejs runtime environment to your cluster
+
+Fission language support is enabled by creating an _Environment_.  An
+environment is the language-specific part of Fission.  It has a
+container image in which your function will run.
+
+```
+fission environment create --name nodejs --image fission/node-env
+```
+
+## Write a simple function in Nodejs
+
+Create a file hello-world.js with the following content
+
+```
+module.exports = async function(context) {
+    return {
+        status: 200,
+        body: "hello, world!\n"
+    };
+}
+```
+
+Create a function with the following command. Note that the value for `--env` flag is `nodejs` which was created with `fission env create` command above.
+
+```
+fission function create --name hello-world --code hello-world.js --env nodejs
+```
+
+Test the function with the below command and you should see "hello, world!" in the output
```

Comment:
Test the function with the command below. If everything succeeds, you should see "hello, world!" as the output.